### PR TITLE
[Meta-Data Update] New Entry - JKs Skyrim

### PR DESCRIPTION
**Pull request by MacSplody**
Compatibility notes - https://www.nexusmods.com/skyrimspecialedition/mods/6289

- Paradise City, versions with changes to Rorikstead are incompatible.
- Bring Out Your Dead.esp compatible when loaded after JKs.
- Open cities & Dawn of Skyrim (Blues Skyrim.esp) are currently incompatible, Patches planned.
- Messages added for provided patches.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -946,6 +946,78 @@ plugins:
...
+      - <<: *hasPluginPatch
+        subs:
+          - 'Holidays'
+          - '[JK''s Skyrim SE Holidays Patch](https://www.nexusmods.com/skyrimspecialedition/mods/6289/)'
+        condition: 'active("Holidays.esp") and not active("JKs Skyrim_Holidays_Patch.esp")'
+      - <<: *hasPluginPatch
+        subs:
+          - 'Leaf Rest'
+          - '[JK''s Skyrim SE Leaf Rest Patch](https://www.nexusmods.com/skyrimspecialedition/mods/6289/)'
+        condition: 'active("LeafRest OnePiece.esp") and not active("JKs Skryim Leafrest Patch.esp")'
```
Comment (MacSplody) [inline masterlist.yaml:1010]: typo is correct esp name "JKs Skryim Leafrest Patch.esp"

File: masterlist.yaml [hunk 2]
```diff
@@ -946,6 +946,78 @@ plugins:
...
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
+  - name: "JKs Skyrim.esp"
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/6289/']
+    inc:
+      - 'Blues Skyrim.esp' # DAWN OF SKYRIM, patch is planned
+      - 'Open Cities Skyrim.esp'
+      - 'Paradise_City_Merged.esp'
+      - 'Paradise_City_MergedNoWindhelm.esp'
+      - 'Paradise_City_MergedSummerWindhelm.esp'
+      - 'Paradise_City_Rorikstead.esp'
```
Comment (MacSplody) [inline masterlist.yaml:961]: Can conditionals be added on inc? 
Have two versions of the same mod, same esp name. One is compatible.
A warning message & checksum comparison will be fine anyway.

Comment (Ortham) [reply masterlist.yaml:961]: Yes, inc entries are [Files](https://loot-api.readthedocs.io/en/latest/metadata/data_structures/file.html).

Comment (MacSplody) [reply masterlist.yaml:961]: Thanks, conditional works fine, display doesn't seem to work though. I'll add an additional warn message instead with link to compatible version.

    inc:
      - name: 'Friendlier Taverns.esp'
        condition: 'active("Friendlier Taverns.esp") and checksum("Friendlier Taverns.esp",0C38A36A)'
        display: '[Friendlier Taverns with Baths and Barns SSE](https://www.nexusmods.com/skyrimspecialedition/mods/1399/)'

Comment (Ortham) [reply masterlist.yaml:961]: Display should, if it doesn't it's a bug, so don't work around it.

Comment (MacSplody) [reply masterlist.yaml:961]: Sorry didn't see comment before change, I have a additional message  which points to compatible version, should still be required. I will add in the display again now.

**Pull request merged**